shenyu-admin/src/main/java/org/apache/shenyu/admin/service/impl/AppAuthServiceImpl.java

```diff
@@ -233,9 +231,9 @@ public ShenyuAdminResult syncData() {
             return ShenyuAdminResult.success();
         }
 
-        Set<String> idSet = appAuthDOList.stream().map(BaseDO::getId).collect(Collectors.toSet());
-        Map<String, List<AuthParamData>> paramMap = this.prepareAuthParamData(idSet);
-        Map<String, List<AuthPathData>> pathMap = this.prepareAuthPathData(idSet);
+        List<String> idList = appAuthDOList.stream().map(BaseDO::getId).collect(Collectors.toList());
```

Comment:
How about change to `List<String> idList = appAuthDOList.stream().map(BaseDO::getId).distinct().collect(Collectors.toList());`?